<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_BelieveOrNot</class>
 <widget class="QWidget" name="Form_BelieveOrNot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>741</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>420</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Load">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Next">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
